spec/hosts/nodes/lsstcam-pcs01.cp.lsst.org_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'lsstcam-pcs01.cp.lsst.org', :sitepp do
+  on_supported_os.each do |os, os_facts|
+    next unless os =~ %r{almalinux-9-x86_64}
+
+    context "on #{os}" do
+      let(:facts) do
+        override_facts(os_facts,
+                       fqdn: 'lsstcam-pcs01.cp.lsst.org',
+                       is_virtual: false,
+                       virtual: 'physical',
+                       dmi: {
+                         'product' => {
+                           'name' => 'UNO-148-B',
+                         },
+                       })
+      end
+      let(:node_params) do
+        {
+          role: 'ccs-hcu',
+          cluster: 'lsstcam-ccs',
+          site: 'cp',
+        }
+      end
+
+      it { is_expected.to compile.with_all_deps }
+
+      include_examples 'baremetal no bmc'
+      include_context 'with nm interface'
+
+      it { is_expected.to have_nm__connection_resource_count(2) }
+
+      context 'with enp1s0' do
+        let(:interface) { 'enp1s0' }
+
+        it_behaves_like 'nm enabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it { expect(nm_keyfile['ipv4']['address1']).to eq('139.229.175.94/26,139.229.175.126') }
+        it { expect(nm_keyfile['ipv4']['dns']).to eq('139.229.160.53;139.229.160.54;139.229.160.55;') }
+        it { expect(nm_keyfile['ipv4']['dns-search']).to eq('cp.lsst.org;') }
+        it { expect(nm_keyfile['ipv4']['method']).to eq('manual') }
```

Comment:
```suggestion
        it_behaves_like 'nm manual interface'
```

Using the shared example makes refactoring slightly easy in the future and it currently test slightly more than this line but the man reason to use the shared example is consistency with the other spec files.